### ISSUE TITLE
Chart question

### ISSUE THREAD

**Issue by panos-stavrianos** [code-and-effect/effective_datatables#120]
Hi everyone, thank you for this awesome library! 
I am trying to display a line chart but it's seems that a bar chart is rendered. So this is what i am doing
```ruby  
class FoodChart < Effective::Datatable
  charts do
    chart :my_chart, 'LineChart', label: 'Chart', legend: false do |collection|
      [['Food', 'Price'],
       ['Bacon', 10],
       ['Eggs', 20],
       ['Toast', 30]]
    end
  end
  collection do
    []
  end
end
```
```ruby
    @food_chart = FoodChart.new
    <%= render_datatable_charts(@food_chart) %>
```
 And this is what i am getting
![chart](https://user-images.githubusercontent.com/20128753/64895486-6a10ef80-d685-11e9-8173-8c4558403067.png)
Any help will be appreciated! Thanks

**Comment by matt-riemer** [comment]
I can point you towards

https://developers.google.com/chart/interactive/docs/gallery/linechart

I haven't used this charting stuff in a very long time, it's possible it's the Google API has changed since this was developed.

Do you feel upto debugging it? It's probably a javascript issue. 

https://github.com/code-and-effect/effective_datatables/blob/master/app/assets/javascripts/effective_datatables/charts.js.coffee

**Comment by matt-riemer** [comment]
Hey!

This was fixed in effective_datatables 4.7.15

**Comment by panos-stavrianos** [comment]
Thank you! It works perfectly!
